plugins/janus_sip.c

```diff
@@ -4762,8 +4762,8 @@ void janus_sip_sofia_callback(nua_event_t event, int status, char const *phrase,
 		}
 		case nua_i_options:
 			JANUS_LOG(LOG_VERB, "[%s][%s]: %d %s\n", session->account.username, nua_event_name(event), status, phrase ? phrase : "??");
-			/* FIXME Should we handle this message? for now we reply with a 405 Method Not Implemented */
-			nua_respond(nh, 405, sip_status_phrase(405), TAG_END());
+			/* Stack responds automatically to OPTIONS request unless OPTIONS is 
+			   included in the set of application methods, set by NUTAG_APPL_METHOD(). */
```

Comment:
Please remove the spaces there, before "included".

Reply:
Sure! Update comment formatting according to other multiline comments.